### DISCUSSION TITLE
Clarification regarding Image size

### DISCUSSION THREAD

**Discussion by jiteshvsk** [Sunbird-cQube/community#751] [Q&A] [answered]
Hi, will require information on the image size for:
a. Logo on login page and for the dashboard
b. Image on login page which comes on right hand side
c. Images for the logos above the space for putting login credentials

**Comment by RNarmatha** [comment, marked as answer]
<html>
<body>
<b style="font-weight:normal;" id="docs-internal-guid-355eea2d-7fff-7529-c3f3-a8046f4b31fe"><div dir="ltr" style="margin-left:0pt;" align="left">

Type of Screen | Width in Pixel | Height in Pixel
-- | -- | --
Larger Screen( TV) | 780 | 600
Laptop Screen | 790 | 630
Medium Device |   |  
Size Above 1360 pixel Below 1520 pixel | 800 | 630
Size Above 1500 pixel Below 1710 pixel | 890 | 740

</div></b>
</body>
</html>

**Comment by RNarmatha** [comment]
Solution Provided
